package.json

```diff
@@ -29,50 +29,39 @@
     "node": ">=4.8.4"
   },
   "scripts": {
-    "test": "grunt test"
+    "clean": "rm -rf package-lock.json && rm -rf node_modules && rm -rf coverage",
+    "test": "mocha --recursive 'test/**/*.js' --reporter spec --timeout 3000 --ui bdd --exit",
+    "test:watch": "npm run test -- -w",
+    "lint": "jshint lib/**/*.js test/**/*.js --config .jshintrc",
+    "test:coverage": "istanbul cover _mocha -- --recursive 'test/**/*.js' --reporter spec --exit",
+    "watch:lib": "watch 'npm run lint && npm test' ./lib"
   },
   "dependencies": {
-    "async": "1.5.2",
-    "body-parser": "~1.11.0",
+    "async": "2.6.1",
+    "body-parser": "~1.18.3",
     "command-shell-lib": "1.0.0",
-    "express": "~4.11.2",
-    "jison": "0.4.17",
-    "logops": "1.0.0",
-    "moment": "~2.19.4",
+    "express": "~4.16.4",
+    "jison": "0.4.18",
+    "logops": "2.1.0",
+    "moment": "~2.22.2",
     "moment-timezone": "~0.5.21",
-    "mongodb": "2.2.35",
-    "mongoose": "4.13.12",
+    "mongodb": "3.1.8",
+    "mongoose": "5.3.6",
     "mu2": "~0.5.20",
-    "mustache": "2.2.1",
+    "mustache": "3.0.0",
     "node-uuid": "~1.4.1",
-    "query-string": "4.3.4",
-    "request": "2.39.0 - 2.81.0",
+    "query-string": "6.2.0",
+    "request": "2.88.0",
     "revalidator": "~0.3.1",
-    "sax": "~0.6.0",
-    "underscore": "~1.7.0",
-    "xmldom": "0.1.19"
+    "underscore": "~1.9.1",
+    "xmldom": "0.1.27"
   },
   "devDependencies": {
-    "chai": "3.5.0",
-    "closure-linter-wrapper": "1.0.1",
-    "grunt": "0.4.5",
-    "grunt-contrib-clean": "0.7.0",
-    "grunt-contrib-jshint": "0.12.0",
-    "grunt-contrib-watch": "0.6.1",
-    "grunt-dox": "0.5.0",
-    "grunt-exec": "0.4.6",
-    "grunt-githooks": "0.5.0",
-    "grunt-github-pages": "0.0.4",
-    "grunt-gjslint": "0.2.1",
-    "grunt-mkdir": "0.1.2",
-    "grunt-mocha-test": "0.12.7",
-    "grunt-plato": "~1.3.0",
-    "istanbul": "~0.1.34",
-    "mocha": "2.4.5",
-    "nock": "10.0.0",
-    "should": "8.4.0",
-    "sinon": "1.17.4",
-    "sinon-chai": "2.8.0",
-    "timekeeper": "0.0.5"
+    "istanbul": "~0.4.5",
+    "mocha": "5.2.0",
+    "nock": "10.0.1",
+    "should": "13.2.3",
+    "timekeeper": "2.1.2",
+    "jshint": "^2.9.6"
```

Comment:
Let's try to avoid ^ dependencies. We can use:

    "jshint": "~2.9.6"

Reply:
Fixed with 955d7b31dfd42cc91ddf36ec34ce8d6d8103bf1b